sklearn_extra/robust/mean_estimators.py

```diff
@@ -97,14 +97,18 @@ def huber(X, c=1.35, T=20):
     X : array like, length = n_sample
         sample from which we want an estimator of the mean
 
-    c : float >0, default = 1.35
+    c : float >0, default = None
         parameter that control the robustness of the estimator.
         c going to zero gives a  behavior close to the median.
         c going to infinity gives a behavior close to sample mean.
+        if c is None, the iqr is used as heuristic.
```

Comment:
Might we worth reminding,

```suggestion
        if c is None, the interquartile range (IQR) is used 
        as heuristic.
```
I didn't understand what irq meant on the first read.